search-sentiments-map/routes/search.js

```diff
@@ -272,11 +274,33 @@ async function addWorldDataByTopicToDatastore(topic, countriesData) {
   try {
     await datastore.save(entity);
     console.log(topic);
-    console.log(`Custom Search Result ${worldDataByTopicKey.id} created successfully.`);
+    console.log(
+        `Custom Search Result ${worldDataByTopicKey.id} created successfully.`
+    );
   } catch (err) {
     console.error('ERROR:', err);
   }
 }
 
-module.exports.router = router;
-module.exports.updateSearchResults = updateSearchResults;
\ No newline at end of file
+/** Sleep for one minute. */
+function sleepForOneMinute() {
```

Comment:
Also how about a generic sleep where you can pass in a param according to the time you need